pandas/tests/window/test_moments.py

```diff
@@ -1891,6 +1891,20 @@ def test_rolling_corr_with_zero_variance(self, window):
 
         assert s.rolling(window=window).corr(other=other).isna().all()
 
+    def test_rolling_max_datetimeindex(self):
```

Comment:
move to tests/window/test_timeseries_window.py and put near similar tests (this might be a duplicate)

Reply:
Thanks for the feedback, @jreback. Moved the test to `tests/window/test_timeseries_window.py`. This test is different by being the only one using minutes as frequency in the file. Other minor differences is that it's using Series as opposed to DataFrame and explicit definition of expected.

Having said that, I agree that there are very similar tests and I'm happy to remove it all together.

Reply:
@ganevgv if you would like to clean that up and parametrize frequencies / containers that would be *very* welcome